docs/_docs/02_testing-best-practices/tools-and-plugins.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: collection-browser-doc
+title: Tools and Plugins
+category: testing-best-practices
+excerpt: >-
+  Additional tools and plugins for terratest to make integration with existing workflows easier.
+tags: ["testing-best-practices", "alternatives","plugins","tooling"]
+order: 214
+nav_title: Documentation
+nav_title_link: /docs/
+---
+
+## Tools and Plugins
+
+Since almost all project has an infrasturcture that needs to be tested, it is great to have different kind of tools and plugins which can help integrating Terratest into your build pipelines seamlessly. Here is a quick list of the different tooling and various plugins out there.
+
+### Terratest Maven Plugin
+
+Terratest Maven Plugin aims to bring the marvelous terratest to the JVM world. Create your Go based tests beside your Java code with Maven and run them together. You can export the results into Json or an HTML page. As the plugin is MIT licenced it is easy and painless to integrate into any Java+Maven combination. To learn more check out the website: [Terratest Maven Plugin](https://terratest-maven-plugin.github.io) and the [GitHub repository](https://github.com/terratest-maven-plugin/terratest-maven-plugin)
```

Comment:
```suggestion
The Terratest Maven Plugin aims to bring Terratest to the JVM world. Create your Go based tests beside your Java code with Maven and run them together. You can export the results into Json or an HTML page. As the plugin is MIT licensed, it is easy and painless to integrate into any Java+Maven combination. To learn more check out the website: [Terratest Maven Plugin](https://terratest-maven-plugin.github.io) and the [GitHub repository](https://github.com/terratest-maven-plugin/terratest-maven-plugin)
```

Reply:
Hi! Thanks! Did the changes, can you please check them again?